msg/sensor_accel_fifo.msg

```diff
@@ -4,10 +4,15 @@ uint64 timestamp_sample
 uint32 device_id          # unique device ID for the sensor that does not change between power cycles
 
 float32 dt                # delta time between samples (microseconds)
-float32 scale
+
+float32 scale             # sensor resolution scale (LSB/g)
+
+float32 dynamic_range     # dynamic range of the sensor (m/s^2)
 
 uint8 samples             # number of valid samples
 
 int16[32] x               # acceleration in the FRD board frame X-axis in m/s^2
 int16[32] y               # acceleration in the FRD board frame Y-axis in m/s^2
 int16[32] z               # acceleration in the FRD board frame Z-axis in m/s^2
+
+uint8 ORB_QUEUE_LENGTH = 4
```

Comment:
Why change ORB_QUEUE_LENGTH? These are pretty large messages, so it ends up being expensive on older boards (and multiple sensors).

Reply:
Modeling the Fifo messages. I can revert that, it's not necessary.

Reply:
Oh, hang on, I think I just made a mistake there.

Reply:
Reverted and rebased against main.